huxley/www/js/components/Shaker.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2011-2017 Berkeley Model United Nations. All rights reserved.
+ * Use of this source code is governed by a BSD License (see LICENSE).
+ */
+
+'use strict';
+
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+var cx = require('classnames');
+
+require('css/Shaker.less');
+
+var Shaker = React.createClass({
+  childContextTypes: {
```

Comment:
I thought about that, but it forces any component to know about its parent components. With this, we break the component abstraction a little with context, but the function isn't required to be passed-in, so the child components still don't force the parents to do anything.